### PR TITLE
Proposed changes from DGIWG P2 panel

### PR DESCRIPTION
**Pull request by gcebelieu**
These are proposed changes made by Ulf Tennfors, aimed at complete file naming scheme according to current version of DGED.

### CODE REVIEW

**Comment by gcebelieu** [comment]
Form @UlfTennfors github account.

File: dem2dged_geo.py [hunk 1]
```diff
@@ -99,10 +136,17 @@ def main(args):
             dl.dp(" ")
             dl.dp("-"*70)
             dl.dp("Creating elevation raster %s" %(namnam))
-            cmdstr = """gdalwarp -t_srs EPSG:%s+3855 -te %s %s %s %s -dstnodata -32767 -tr %s %s -r cubic -co COMPRESS=LZW --config GTIFF_REPORT_COMPD_CS YES %s %s""" %(my_out_srs, minlon, minlat, maxlon, maxlat, lonres, latres, pargs.input_raster, namnam )
+            # EPSG:3855 gives the vertical reference of EGM2008. For a lot of files, gdalwarp doesn't work when this reference is given.
+#            cmdstr = """gdalwarp -t_srs EPSG:%s+3855 -te %s %s %s %s -dstnodata -32767 -tr %s %s -r cubic -co COMPRESS=LZW --config GTIFF_REPORT_COMPD_CS YES %s %s""" %(my_out_srs, minlon, minlat, maxlon, maxlat, lonres, latres, pargs.input_raster, namnam )
```
Comment (lethorable) [inline dem2dged_geo.py]: Suggest to delete this line in order to clean up comments

File: dem2dged_utm.py [hunk 2]
```diff
@@ -6,17 +6,21 @@
 import datetime
 import dem2dged_lib as dl
 
-debug = False
+debug = True
```
Comment (lethorable) [inline dem2dged_utm.py]: Should be "false" in production

**Review by lethorable — CHANGES_REQUESTED** [review]
Thanks for the excellent work. I have two suggestions for changes. Kind regards, Thorbjørn

File: dem2dged_geo.py (hunk 1 above)
Comment (gcebelieu) [reply dem2dged_geo.py]: We agree. We will remove these lines

File: dem2dged_utm.py (hunk 2 above)
Comment (gcebelieu) [reply dem2dged_utm.py]: Agree also.

File: dem2dged_geo.py (hunk 1 above)
Comment (gcebelieu) [reply dem2dged_geo.py]: It seems that the commented lines have already been removed.

**Pull request merged**